### PR TITLE
[docs] Vue.js in Gitpod

### PR DESCRIPTION
**Pull request by JesterOrNot**
See #383

### CODE REVIEW

File: src/docs/vue-in-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+```
+> Please note that if you already have a [`.gitpod.yml`](https://www.gitpod.io/docs/config-gitpod-file/) and a [`.gitpod.Dockerfile`](https://www.gitpod.io/docs/config-docker/) you just need to add the above command don't worry about the [`gp init`](https://www.gitpod.io/docs/command-line-interface/#init) part
+
+## VSCode Extensions 
+
+### Vetur
+
+![Vetur extension](images/Vetur.png)
+
+Vetur provides syntax highlighting, snippets, Emmet support, lintiing/error checking, auto formatting, and auto-complete for Vue files.
```
Comment (svenefftinge) [inline src/docs/vue-in-gitpod.md]: linting

File: src/docs/vue-in-gitpod.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+To add this extension to a new repository open the project in Gitpod and in the terminal run [`gp init`](https://www.gitpod.io/docs/command-line-interface/#init)
```
Comment (svenefftinge) [inline src/docs/vue-in-gitpod.md]: I don't think this should be repeated over and over again. I'd rather prefer to say.
```md
To install the Vetur extension add the following to your [`.gitpod.yml`](https://www.gitpod.io/docs/config-gitpod-file/):
```

File: src/docs/vue-in-gitpod.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+</div>
+
+## Vue-CLI
+
+To install the Vue.js command line interface in your current workspace run the following
+```bash
+npm i -g vue-cli
+```
+
+To install globally across all workspaces first if you haven't already run [`gp init`](https://www.gitpod.io/docs/command-line-interface/#init) which should generate two files [`.gitpod.yml`](https://www.gitpod.io/docs/config-gitpod-file/) and [`.gitpod.Dockerfile`](https://www.gitpod.io/docs/config-docker/) inside the [`.gitpod.Dockerfile`](https://www.gitpod.io/docs/config-docker/) add the following
```
Comment (svenefftinge) [inline src/docs/vue-in-gitpod.md]: see comment below. I'd rather explain .gitpod.yml and how to create it in one place.

File: src/docs/vue-in-gitpod.md (hunk 1 above)
Comment (JesterOrNot) [reply src/docs/vue-in-gitpod.md]: Thanks!

File: src/docs/vue-in-gitpod.md (hunk 3 above)
Comment (JesterOrNot) [reply src/docs/vue-in-gitpod.md]: True we should probably follow the D.R.Y. principle here. :smile:

**Comment by JesterOrNot** [comment]
Sorry I meant to resolve it not delete it

**Comment by JesterOrNot** [comment]
I did fix it though so that is good!

**Comment by JesterOrNot** [comment]
Thanks, Merging :rocket:

**Pull request merged**